docs/self-hosted-appcircle/install-server/helm-chart/upgrades.md

```diff
@@ -17,7 +17,25 @@ helm upgrade appcircle-server appcircle/appcircle \
 ```
 
```

Comment:
It will be good to add command sample to see the current version of Appcircle deploy.